docs/installation/arch.md

```diff
@@ -50,7 +54,30 @@ Please make sure you have followed [Install dependencies](#install-dependencies)
 
 The steps to install from source can be found [here](source.md)
 
-### Launch the Streamdeck UI
+### Install with the AUR
+
+The AUR currently has an [easy install script](https://aur.archlinux.org/packages/streamdeck-ui) that is being maintained by dhtseany.
+> Additional packages may be required before the script will finish, once those have been installed the script will finish installing.
+
+``` bash
+cd Downloads
+git clone https://aur.archlinux.org/streamdeck-ui.git
+cd streamdeck-ui
+makepkg
+sudo pacman -U streamdeck-ui-3.1.0-1-any.pkg.tar.zst
+```
+
+### Install with yay
+
+[yay is a Pacman wrapper and AUR helper](https://aur.archlinux.org/packages/yay) that will download and install all of the prerequisites for you, along with the StreamDeck UI app itself.
+
+``` bash
+yay -S streamdeck-ui
+```
+
+> For both yay and the AUR you can also install the [Development branch](https://aur.archlinux.org/packages/streamdeck-ui-develop) if you want to stay up to date with improvements.
```

Comment:
Guess this bit should be a bit more clear. 
Development doesn't really mean that its to stay up to date, lets call it bleeding edge. 

streamdeck-ui is Stable release, new features will only drop here after a bit on the development branch, and for now l am updating this one every month or every 2 months depending. 
streamdeck-ui-develop, is unstable and under development for those who want to assist with testing or use the latest and greatest even if it means the application might break.

Reply:
Updated it to make a little more sense, hopefully that keeps it clear and concise.